components/Mitra/index.vue

```diff
@@ -166,6 +246,91 @@ export default {
         this.query.order_by = 'users.created_at'
         this.query.sort_by = 'desc'
       }
+    },
+    verifyUser (item) {
+      this.dataMitra = item
+      const { name, partner } = item || null
+      this.contentMitra.data = partner?.name ? `${name} - ${partner.name}` : name
+      this.contentMitra.header = 'Verifikasi Mitra'
+      this.contentMitra.title = 'Apakah anda yakin ingin menerima mitra ini?'
+      this.showDialogVerify = true
+    },
+    onRejectData () {
+      const { email } = this.dataMitra || null
```

Comment:
same as the comment before

Reply:
I already replied this above, and changes below, thanks @Ibwedagama